src/SecTester.Scan/CI/CiServer.cs

```diff
@@ -2,5 +2,50 @@ namespace SecTester.Scan.CI;
 
 public record CiServer(string? ServerName = default)
 {
-  public static readonly CiServer? Unknown;
+  public static CiServer APPCENTER { get; } = new("Visual Studio App Center");
```

Comment:
Please follow the naming convention, instead of using a property name introduce an `id` or `value` field. Use it to filter enumeration.

```suggestion
  public static CiServer AppCenter { get; } = new("Visual Studio App Center");
```